pkgs/development/tools/misc/debugedit/default.nix

```diff
@@ -0,0 +1,30 @@
+#TODO@deliciouslytyped The tool seems to unnecessarily force mutable access for the debugedit `-l` feature
+{fetchgit, lib, stdenv, autoreconfHook, pkg-config, elfutils, help2man, util-linux}:
+stdenv.mkDerivation {
+  name = "debugedit";
+  version = "unstable-2021-07-05";
```

Comment:
There was a release in june: https://sourceware.org/git/?p=debugedit.git;a=tag;h=97ee9665f8d401d52340845749728ebb7c482a79

any reason to use current master?

Reply:
I intend to contact upstream about some usability fixes that I would expect to get in relatively quickly.

Reply:
fine then :)